.travis.yml

```diff
@@ -38,15 +38,11 @@ env:
   - UNIT_TEST="1" ODOO_REPO="OCA/OCB" INCLUDE="base_url"
 
 install:
-  - git clone -b configure_line_with_error https://github.com/akretion/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
+  - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
   - pip install pylint==1.8.4
   - printf '[options]\n\nrunning_env = dev\nkeychain_key_dev = HRdthrkXp28SCHX0kmwOgIob-82N-w8zWoE_UsctE7E=' > ${HOME}/.openerp_serverrc
-  - git clone https://github.com/akretion/akretion-quality-tools ${HOME}/akretion-quality-tools
-  - export PATH=${HOME}/akretion-quality-tools/travis:${PATH}
-  - export PATH=.travis:${PATH}
```

Comment:
This `.travis` in PATH must stay, because that is where `travis_delete_algolia_index` is.